gridt/src/app/about/tutorial/tutorial.page.scss

```diff
@@ -0,0 +1,45 @@
+.intro-logo {
+    max-width: 50vw;
+    max-height: 50vw;
+    overflow: hidden;
+}
+
+.intro-img {
+    max-width: 90vw;
+    max-height: 90vw;
+    overflow: hidden;
+}
+
+p {
+    padding: 1em;
+}
+
+.step-one {
+    height: 100vh;
+    flex-direction: column;
+    text-align: justify;
+}
+
+.step-two {
+    height: 100vh;
+    flex-direction: column;
+    text-align: justify;
+}
+
+.step-three {
+    height: 100vh;
+    flex-direction: column;
+    text-align: justify;
+}
+
+.step-four {
+    height: 100vh;
+    flex-direction: column;
+    text-align: justify;
+}
+
+.step-five {
```

Comment:
This class seems to be the same for each step. You could make a single class named `.step` and add that to each step in the html.

Reply:
Sharp. They were different colors before, but that didn't look good with the animations. So now it is indeed just one style.